emf/complete/src/main/java/powsybl/tutorials/emf/BalancesAdjustmentValidationParameters.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package powsybl.tutorials.emf;
+
+import com.powsybl.commons.config.ModuleConfig;
+import com.powsybl.commons.config.PlatformConfig;
+
+import java.util.*;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public final class BalancesAdjustmentValidationParameters {
```

Comment:
Add javadoc on this class or comments to explain what it is used for and how to use it. This should also be developed more at length on the website.